### ISSUE TITLE
Valid YouTube Playlist Causing Error

### ISSUE THREAD

**Issue by Pi-Man** [sedmelluq/lavaplayer#591]
A playlist I've been using to collect songs on youtube is suddenly not working, it was working a few weeks ago but not now.  I tried updateing to latest lavaplayer (1.3.72) and it still is not working.  Other smaller playlists work just fine including playlists with deleted songs.  The playlist that is not working is 734 songs so I'm speculating that it is the length of the playlist, but I don't have any other long playlists to test.  (the second longest playlist I have is 23 songs and works fine)

this is the link to the playlist that is not playing <https://www.youtube.com/playlist?list=PLi9EN2LtHhO2AIKkRNaoPbbDNiSNOuamJ>

and here is the stack trace
```
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something went wrong when looking up the track
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:44)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.dispatchItemLoadFailure(DefaultAudioPlayerManager.java:236)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:225)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
	Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation: 
  lavaplayer.version: 1.3.72
  os.arch: amd64
  os.name: Windows 10
  os.version: 10.0
  java.vendor: Oracle Corporation
  java.version: 1.8.0_191
  java.runtime.version: 1.8.0_191-b12
  java.vm.version: 25.191-b12
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: java.io.IOException: Invalid status code for playlist response: 404
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:58)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.playlist(YoutubeAudioSourceManager.java:243)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.playlist(YoutubeAudioSourceManager.java:228)
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.routeFromMainDomain(DefaultYoutubeLinkRouter.java:70)
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.route(DefaultYoutubeLinkRouter.java:42)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:196)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:131)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:436)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:418)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:217)
	... 4 more
Caused by: java.lang.RuntimeException: java.io.IOException: Invalid status code for playlist response: 404
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubePlaylistLoader.load(DefaultYoutubePlaylistLoader.java:47)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.playlist(YoutubeAudioSourceManager.java:240)
	... 12 more
Caused by: java.io.IOException: Invalid status code for playlist response: 404
	at com.sedmelluq.discord.lavaplayer.tools.io.HttpClientTools.assertSuccessWithContent(HttpClientTools.java:155)
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubePlaylistLoader.buildPlaylist(DefaultYoutubePlaylistLoader.java:100)
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubePlaylistLoader.load(DefaultYoutubePlaylistLoader.java:45)
	... 13 more

```

**Comment by llacb47** [comment]
This API endpoint no longer exists and needs to be fixed. 

https://github.com/sedmelluq/lavaplayer/blob/3d54ef10bdf6b7da32fe08beff761a8efe560cbb/main/src/main/java/com/sedmelluq/discord/lavaplayer/source/youtube/DefaultYoutubePlaylistLoader.java#L179

**Comment by Pi-Man** [comment]
I figured that was probably the issue, do I just have to wait for an update then?

**Comment by MCOfficer** [comment]
Yes

> I figured that was probably the issue, do I just have to wait for an update then?

Yup.

**Comment by sedmelluq** [comment]
Fixed in `1.3.73`.
